barretenberg_wrapper/src/pedersen.rs

```diff
@@ -40,7 +40,7 @@ pub fn encrypt(inputs_buffer: &[[u8; 32]]) -> ([u8; 32], [u8; 32]) {
     let buffer_len = inputs_buffer.len() as u32;
     let mut result = [0_u8; 64];
     buffer.extend_from_slice(&buffer_len.to_be_bytes());
-    for e in &*inputs_buffer {
+    for e in inputs_buffer {
```

Comment:
Is this correct? I just did what clippy said

Reply:
Same comment as here: https://github.com/noir-lang/aztec-connect/pull/14/files#r1067451294